pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -243,6 +243,20 @@ public static class Broker {
         Math.max(1, Math.min(10, Runtime.getRuntime().availableProcessors() / 2));
     // Same logic as CombineOperatorUtils
 
+    // Config for Jersey ThreadPoolExecutorProvider.
+    // By default, Jersey uses the default unbounded thread pool to process queries.
+    // By enabling it, BrokerManagedAsyncExecutorProvider will be used to create a bounded thread pool.
+    public static final String CONFIG_OF_ENABLE_BOUNDED_THREAD_POOL = "pinot.broker.enable.bounded.threadPool";
+    public static final boolean DEFAULT_ENABLE_BOUNDED_THREAD_POOL = false;
+    // Default capacities for the bounded thread pool
+    public static final String CONFIG_OF_MAX_POOL_SIZE = "pinot.broker.max.pool.size";
```

Comment:
Config name can be changed to: `pinot.broker.async.executor.max.pool.size` (similarly for other configs).

Since broker uses a few other thread-pools it's best to be explicit.

Reply:
Thanks Ankit, addressed.